tasks/cluster_resource.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Ensure pcs_resource are present.
+  pcs_resource:
+    name: "{{ item.name }}"
+    state: "{{ item.state | default(omit) }}"
+    resource_class: "{{ item.resource_class | default(omit) }}"
+    resource_type: "{{ item.resource_type | default(omit) }}"
+    options: "{{ item.options | default(omit) }}"
+    force_resource_update: "{{ item.force_resource_update | default(omit) }}"
+    cib_file: "{{ item.cib_file | default(omit) }}"
+    child_name: "{{ resitemource.child_name | default(omit) }}"
```

Comment:
(medium) - `resitemource.child_name` is not defined, most probably this is `item.child_name`

Reply:
Sorry for that must be fixed by the commit 76e6c8f

Reply:
also ok here.